java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/gradle/GradleUtils.java

```diff
@@ -200,7 +200,7 @@ public static boolean isInBuildGradle(Path path, CharSequence csq) {
                         } catch (IOException e) {
                             return Stream.empty();
                         }
-                    }).anyMatch(s1 -> s1.contains(csq));
+                    }).anyMatch(s1 -> s1.matches(regex));
```

Comment:
Shouldn't this be using `Matcher.find()` instead of matches, to remove the need for a `.*` at the start and end? Also I would suggest using `java.util.regex.Pattern#MULTILINE` rather than iterating over the lines, so you can match multi line constructs.

Reply:
How's 3f7f8b8?